mirrord/config/src/feature/copy_target.rs

```diff
@@ -0,0 +1,79 @@
+//! Config for the `copy target` feature. [`CopyTargetFileConfig`] does follow the pattern of other
+//! [`feature`](crate::feature) configs by not implementing
+//! [`MirrordToggleableConfig`](crate::util::MirrordToggleableConfig). The reason for this is that
+//! [`ToggleableConfig`](crate::util::ToggleableConfig) is enabled by default. This config should be
+//! disabled unless explicitly enabled.
+
+use mirrord_analytics::CollectAnalytics;
+use schemars::JsonSchema;
+use serde::Deserialize;
+
+use crate::config::{ConfigContext, FromMirrordConfig, MirrordConfig, Result};
+
+/// Allows the user to target a pod created dynamically from the orignal [`target`](#target).
+/// The new pod inherits most of the original target's specification, e.g. labels.
+///
+/// ```json
```

Comment:
Let's maybe also add a `"copy_target": true` example somewhere?
It's good to have this example with all the features, but we should also document that `copy_target` can accepts booleans.